sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -47,30 +38,33 @@ export interface SchemaId {
     id: string;
     location: string;
     locationById: string;
+    serializationType: string;
     version: number;
 }
 
 // @public
-export interface SchemaIdResponse extends SchemaId, Response {
+export interface SchemaRegistry {
+    getSchemaById(id: string, options?: GetSchemaByIdOptions): Promise<Schema>;
+    getSchemaId(schema: SchemaDescription, options?: GetSchemaIdOptions): Promise<SchemaId>;
+    registerSchema(schema: SchemaDescription, options?: RegisterSchemaOptions): Promise<SchemaId>;
 }
 
 // @public
-export class SchemaRegistryClient {
+export class SchemaRegistryClient implements SchemaRegistry {
     constructor(endpoint: string, credential: TokenCredential, options?: SchemaRegistryClientOptions);
     readonly endpoint: string;
-    getSchemaById(id: string, options?: GetSchemaByIdOptions): Promise<SchemaResponse>;
-    getSchemaId(schema: SchemaDescription, options?: GetSchemaIdOptions): Promise<SchemaIdResponse>;
-    registerSchema(schema: SchemaDescription, options?: RegisterSchemaOptions): Promise<SchemaIdResponse>;
+    // (undocumented)
```

Comment:
missing reference docs?

Reply:
This was due to my use of `@inheritdoc`. Fixed.